pkg/koordlet/resourceexecutor/executor.go

```diff
@@ -69,7 +90,14 @@ func (e *ResourceUpdateExecutorImpl) Update(cacheable bool, resource ResourceUpd
 		}
 		return e.updateByCache(resource)
 	}
-	return true, e.update(resource)
+
+	if err := e.update(resource); err == nil {
+		executorMessage := fmt.Sprintf("successfully set resource: %s to value: %s", resource.Key(), resource.Value())
+		e.eventRecorder.Eventf(e.node, corev1.EventTypeNormal, reason.SetCPUSetCPUS, executorMessage)
```

Comment:
It might broadcast too many events at the node, though the events are not complete since the cgroup could be updated in the `updateByCache` and the `UpdateBatch`.

Reply:
`It might broadcast too many events at the node.`  
Is here only need to record event when cgroups resource first set? 
while I cannot find where the  cgroups resource first set, would you give some tips ?  it's would be helpful.

Reply:
OK, changed it to only record event when inject resource change from runtimehooks,  where I guess do the init of the pod cgourp resources.